### ISSUE TITLE
Sales Forecasting Prediction.

### ISSUE THREAD

**Issue by Aaeid123** [Aaeid123/Aaeid-Lightwala#1]
import numpy as np
import pandas as pd
import matplotlib.pyplot as plt
from sklearn.model_selection import train_test_split
from sklearn.linear_model import LinearRegression

data = pd.read_csv('sales_data_sample.csv')

x = data['ORDERNUMBER'].values.reshape(-1,1)
y = data['QUANTITY'].values

x_train, x_test, y_train, y_test = train_test_split(x,y,test_size=0.2,random_state=42)

model = LinearRegression()
model.fit(x_train,y_train)

predict_value = model.predict([[100]])

print("Predicted quantity for 100 sales: ",predict_value)

plt.scatter(x,y,color='red',label='Actual data')
plt.plot(x,model.predict(x), color='blue',label='Fitted Line')
plt.xlabel('ORDERNUMBER')
plt.ylabel('QUANTITY')
plt.title('Sales data predicted')
plt.legend()
plt.show()``
